cropper/src/main/java/com/canhub/cropper/CropImageView.kt

```diff
@@ -1090,7 +1090,7 @@ class CropImageView @JvmOverloads constructor(context: Context, attrs: Attribute
                     val key = state.getString("LOADED_IMAGE_STATE_BITMAP_KEY")
                     if (key != null) {
                         val stateBitmap =
-                            if (BitmapUtils.mStateBitmap != null && BitmapUtils.mStateBitmap.first == key) BitmapUtils.mStateBitmap.second.get() else null
+                            if (BitmapUtils.mStateBitmap != null && BitmapUtils.mStateBitmap!!.first == key) BitmapUtils.mStateBitmap!!.second.get() else null
```

Comment:
let's create a new val
`val stateBitmap = BitmapUtils.mStateBitmap` so we do not need to use `!!` and can use `?`

Reply:
Good idea... : )